### ISSUE TITLE
--meta-subtitle is not written if file does contain any metadata

### ISSUE THREAD

**Issue by scapix** [sandreas/tone#46]
remove all tags (eg. using eyeD3 --remove-all x.mp3)

tone tag x.mp3 --meta-subtitle="Subtitle"

no subtitle tag is written

**Comment by sandreas** [comment]
Thanks for reporting, I'll take a look asap.

**Comment by sandreas** [comment]
Ah, I see. This really is an issue. The reason is, that `Subtitle` is considered as an `AdditionalField` internally and these are only set, if there is a specific set of metadata in the file. 

If all metadata is gone, the format (`id3v1` / `id3v2`) is not detected correctly and so the value is skipped with no message.

As a workaround, you can set ANY non-`AdditionalField` and then try to set `Subtitle` again, e.g.:

```bash
tone tag --meta-title="a title" x.mp3
tone tag --meta-subtitle="a subtitle" x.mp3
```

I'll try to fix that in the next release.

**Comment by sandreas** [comment]
Waiting for response on this: https://github.com/Zeugma440/atldotnet/issues/286#issuecomment-2452417737

If this goes through, I'm able to fix this pretty quick. Otherwise I have to add a workaround.

**Comment by sandreas** [comment]
Should be fixed in `0.1.9`
